pplbench/main.py

```diff
@@ -142,6 +142,10 @@ def read_config(args: Optional[List[str]]) -> SimpleNamespace:
     parser = ArgumentParser()
     parser.add_argument("config", action=ActionJsonSchema(schema=SCHEMA), help="%s")
     config = parser.parse_args(args).config
+    if isinstance(config, dict):
+        from unittest import mock
```

Comment:
Could you add a newline below this import to make the linter happy?

Reply:
Moved the import to the top.